test/parallel/test_tensorflow2_keras_process_sets.py

```diff
@@ -1,21 +1,22 @@
-"""Tests for horovod.tensorflow.keras in TensorFlow 2 that add/remove process sets after initialization.
+"""Tests for horovod.tensorflow.keras in TensorFlow 2 using multiple process sets.
 
 With TensorFlow 2.9 and MPI the option HOROVOD_DYNAMIC_PROCESS_SETS has been observed to cause significant
-slowdowns in all Horovod operations, especially on GPU-equipped AWS instances. For that reason we separate
-out tests that depend on that setting to this script.
+slowdowns in all Horovod operations, especially on GPU-equipped AWS instances. For that reason we collect
+tests for multiple process sets in this script that initializes Horovod with static process sets.
 """
 
 import tensorflow as tf
 import os
+import sys
 import warnings
-
 from distutils.version import LooseVersion
-
 import pytest
-
 from tensorflow import keras
 
+sys.path.append(os.path.join(os.path.dirname(__file__), os.pardir, 'utils'))
```

Comment:
Is this still needed?

Reply:
It isn't. Thanks for pointing that out!

I've cleaned up the imports and `sys.path` manipulation in these test scripts and removed one extra `os.environ["HOROVOD_DYNAMIC_PROCESS_SETS"] = "1"` that fell through the cracks before, also rebased to master.